exporters/autoexport/metrics.go

```diff
@@ -94,4 +99,47 @@ func init() {
 	RegisterMetricReader("none", func(ctx context.Context) (metric.Reader, error) {
 		return newNoopMetricReader(), nil
 	})
+	RegisterMetricReader("prometheus", func(ctx context.Context) (metric.Reader, error) {
+		host := os.Getenv("OTEL_EXPORTER_PROMETHEUS_HOST")
+		if host == "" {
+			host = "localhost"
+		}
+
+		port := os.Getenv("OTEL_EXPORTER_PROMETHEUS_PORT")
+		if port == "" {
+			port = "9464"
```

Comment:
Where does this port come from?

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/082207b76839e81b13e111588c4b2279c41af1a9/specification/configuration/sdk-environment-variables.md?plain=1#L208